data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
 		dialog
 			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
 		"reputation: Hai" += 10
+
+
+
+mission "Returning Home"
+	description "Bring Elliot home to his family on <destination>. Elliot ran away from his family as a teenager and hasn't seen them in almost six years."
+	minor
+	source
+		government "Hai"
+	destination
+		attributes "dirt belt"
+		attributes "farming"
+	passengers 1
+	to offer
+		random < 40
+	on offer
+		conversation
+			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
+			choice
+				`	"Yes, I am."`
+					goto outsider
+				`	"What do you mean 'the outside?'"`
+			
+			`	"Outside of Hai space. Your ship looks newer than most of the ships that humans fly around here."`
+			choice
+				`	"Oh, yes. I do come from outside."`
+			
+			label outsider
+			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
+			`	"Can you take me to <destination>?" Elliot asks.`
+			`	"That's all the way in the Dirt Belt," you say. "Why do you want to go that far?"`
+			`	Elliot looks down at the ground for a second and takes a deep breath while gripping his hat in his hands. "You see, I haven't lived here all my life. I was born on <planet> to a poor farming family. Most of my days were spent working on the farm, which I hated with a passion. One day I just got fed up with things and I ran away.`
+			`	"I ran away from home to the spaceport city and snuck my way into the cargo hold of a freighter. The captain didn't open the cargo hold for three weeks while I stayed in it. Lucky for me the freighter was transporting food, so I had plenty to eat.`
+			`	"When the cargo hold doors finally opened, I ran out as fast as I could, only to run back in at the sight of a group of Hai. I had no idea where I was at the time, but I've been living here ever since. That was when I was sixteen, nearly six years ago now."`
```

Comment:
I have to agree with others that making Elliot 22 seems a bit awkward given the simplistic and child-like way he talks. Maybe improve the sophistication of his dialogue a bit, or reduce his current age by a few years (either by making him younger at the time when he ran away, or reducing the length of his stay in Hai space).

Reply:
I'll make him younger when he ran away and reduce the stay in Hai space a bit, as I think I enjoy the dialogue how it is. I'm sure Elliot is an awkward person to begin with though, as he raised himself for six years starting as a teenager. :)
How does reducing his age when he ran by three years and reducing the stay by one sound? That would me he ran away at thirteen and wants to return at eighteen. That way he's still running away as a teenager but he's returning at a time where he could reasonably still talk in a childish way (I've met some pretty childish 18 year olds), especially having been a rebel for five years.